sympy/polys/polyclasses.py

```diff
@@ -2104,7 +2109,9 @@ def _lcm(f, g):
 
         l = f._mul(g)._exquo(f._gcd(g))
 
-        if l.LC() < 0:
+        if l.dom.is_Field:
+            l = l.monic()
+        elif l.LC() < 0:
             l = l.neg()
```

Comment:
The convention in sympy is that gcd/lcm should be monic if the ground domain is a field e.g.:
```
In [16]: lcm(4*y*x + 1, 4*y*x + 1, y, field=True)
Out[16]:
     1
y + ───
    4⋅x
```
The code to handle this with flint for `QQ[x]` didn't handle this correctly but it fixed here.